packages/@lwc/module-resolver/README.md

```diff
@@ -12,7 +12,7 @@ npm install --save-dev @lwc/module-resolver
 
 ### `resolveModule(specifier, importer, options)`
 
-Synchronously resolve an LWC module from its specifier.
+Synchronously resolve an LWC module from its specifier from an importer path.
```

Comment:
Instead of "resolves" can we use another word or provide a sentence saying what "resolving" is? Is it locating the file and assigning it to the specifier?